.travis.yml

```diff
@@ -42,6 +42,7 @@ matrix:
       env:
         - KAFKA_ADDRS=127.0.0.1:9092
         - NATS_SERVERS=127.0.0.1:4222
+        - SPEC_OPTS="--tag kafka --tag stan"
```

Comment:
The idea is to exclude tags, not include them. we should be able to run all tests on Travis, no need to specify tags (as this will exclude everything without one). On BFS, we exclude GS + S3 as I don't want to run them on Travis - https://github.com/bsm/bfs.rb/blob/master/.travis.yml#L14